auth/src/main/java/com/firebase/ui/auth/ui/AcquireEmailHelper.java

```diff
@@ -80,30 +80,22 @@ private void startEmailHandler(String email, List<String> providers) {
             mActivityHelper.startActivityForResult(registerIntent, RC_REGISTER_ACCOUNT);
         } else {
             // account does exist
-            for (String provider : providers) {
```

Comment:
`List<String> providers` so isn't the first item in the for loop always going to be at index 0?